### PR TITLE
Feature/users country sort

### PR DESCRIPTION
**Pull request by LanesGood**
Initial progress on users table country sort

To do:
- [x] check stub tests for accuracy
- [x] test filter in action with user data

### CODE REVIEW

**Comment by LanesGood** [comment]
@kamicut this is ready for review. Can you check that the tests in `api/tests/api/users.test.js` are correct?

File: api/tests/api/users.test.js [hunk 1]
```diff
@@ -135,6 +135,34 @@ test('Sort users alphabetically z-a', async (t) => {
...
+})
+
+test('Sort users by country alphabetically z-a', async (t) => {
+  const response = await request(app)
+    .get('/scoreboard/api/users/stats/?q=&page=1&sortType=Countries Z-A&active=false')
+    .expect(200)
+
+  const users = await db('users')
+    .select('country')
+    .orderBy('country', 'desc')
```
Comment (kamicut) [inline api/tests/api/users.test.js]: In this and the previous test, you have to make it orderByRaw so it matches

Comment (LanesGood) [reply api/tests/api/users.test.js]: @kamicut thanks, this has been updated

**Comment by LanesGood** [comment]
@kamicut I've updated this to enable the requested featue of a non-sortable Users table. In doing so I uncovered that our nonSortable prop wasn't affecting the tables as intended. This is now working as expected for Users + Countries tables. Campaign tables are still being sorted as expected.

**Pull request merged**